.local/render-combined.py

```
@@ -0,0 +1,107 @@
+#!/usr/bin/env python3
+"""render-combined.py — overlay multiple proto-sweep results into one
+collapse-curve chart. Pass sweep dirs as args; results merged by mult.
+"""
+import sys, os, re, glob
+import matplotlib
+matplotlib.use('Agg')
+import matplotlib.pyplot as plt
+
+dirs = sys.argv[1:] if len(sys.argv) > 1 else sorted(glob.glob('/tmp/proto-sweep-2*'))
```

Ruff (0.15.12)

[error] Probable insecure usage of temporary file or directory: "/tmp/proto-sweep-2*"

(S108)